pocs/utils/logger.py

```diff
@@ -1,16 +1,43 @@
+import os
+import sys
+import time
 import datetime
 import logging
 import logging.config
-import os
-import time
-
+from tempfile import gettempdir
 
 from .config import load_config
 
 
+class PanLogger(object):
+    """ Logger for PANOPTES with format style strings """
+
+    def __init__(self, logger):
+        super(PanLogger, self).__init__()
+        self.logger = logger
+
+    def _process_str(self, fmt, *args, **kwargs):
```

Comment:
I don't understand why this method is necessary. According to the [python documentation](https://docs.python.org/3/library/logging.html#logging.Logger.debug), the loggers do exactly this formatting.

Reply:
This supports the actual format style, e.g. `{:0.2f}`, `{:40s}`, rather than just supporting `{}` as placeholders. The loggers support this via the `%f` syntax, which is simply for backwards-compatibility.  This allows us to just get rid of actual `format` calls in the logger but still keep the same syntax.

I could definitely be wrong about this but the format specifiers didn't seem to work for me otherwise.

Reply:
OK. Perhaps add a comment explaining that?

Reply:
Update now that I'm trying to use the logger: it is confusing to have PanLogger use a different formatting convention from python's loggers, even though the API is otherwise identical.

Reply:
Not sure I follow. You should be able to use either convention.